### ISSUE TITLE
Bundle Minification

### ISSUE THREAD

**Issue by DmitriyGolub** [minzhi3/playable-build-tool#1]
Hey! Can u add minifications settings for cc.js file, maybe minify png by TinyPng. Its really cool bundling in one file, but lib without anything for 3D 2.4 mb hard usefull for playable Ads. When assets bundled as base64 they maximized ended size. It really can help! :) Thanks for that work!

**Comment by minzhi3** [comment]
Thank you for the suggestion. I am working on gzip encoding to reduce size of base64.
In fact for the image files, there are many other plugins which can provide more minify options. You can do this before playable building to get smaller file size. Most ad platforms accept files up to 5MB so in my experience it is enough in most cases

**Comment by DmitriyGolub** [comment]
yep for 2D its awesome. For 3D need phys renderer,  and some modules who weight 1 mb + 1.3mb engine. Thank u for working!
